### DISCUSSION TITLE
Please direct your message hangs forever

### DISCUSSION THREAD

**Discussion by AndrewShear** [Byron/google-apis-rs#520] [Q&A]
When trying to use the secretsmanager1 I ran into a problem which I cannot proceed. I do have ADC setup with through gcloud and I would expect more info. I read through the documentation and some source code but I seem to be stuck. I'm not sure if this is a bug or failure on my part. Here is how to reproduce:

## code:
```rust
extern crate google_secretmanager1 as secretmanager1;
use secretmanager1::{hyper, hyper_rustls, oauth2, SecretManager};
use std::default::Default;

#[tokio::main]
async fn main() {
    let secret: oauth2::ApplicationSecret = Default::default();
    let auth = oauth2::InstalledFlowAuthenticator::builder(
        secret,
        oauth2::InstalledFlowReturnMethod::HTTPRedirect,
    )
    .build()
    .await
    .unwrap();

    let hub = SecretManager::new(
        hyper::Client::builder().build(
            hyper_rustls::HttpsConnectorBuilder::new()
                .with_native_roots()
                .unwrap()
                .https_or_http()
                .enable_http1()
                .build(),
        ),
        auth,
    );

    let result = hub
        .projects()
        .secrets_versions_access("projects/PROJECTID/secrets/SOMESECRET/versions/latest")
        .doit()
        .await;
    println!("{:?}", result);
}
```

however when i run this i get stuck forever with this message in the terminal:
```
Please direct your browser to ?scope=https://www.googleapis.com/auth/cloud-platform&access_type=offline&redirect_uri=http://127.0.0.1:42629&response_type=code&client_id= and follow the instructions displayed there.
```
when attempting to go to this site I do not get any info.

**Comment by Byron** [comment]
It seems the base-URL is missing for the authentication URL, so nothing would happen if a browser see this URL.
Why that is I don't know, except that maybe the respective field in the JSON description of the API is empty?

There are also [other authentication flows](https://docs.rs/yup-oauth2/latest/yup_oauth2/index.html) that can be enabled, maybe that can make a difference here.
